wiki/entry_hahn.qmd

```diff
@@ -0,0 +1,32 @@
+
+When talking about the theoretical arguments put forward in this paper, Reinecke and Meier base their arguments on different theories such as self determination theory. This theory is a common explanation for autonomy needs in digital communication and creates a relationship between different studies. The paper ´Motives for sport participation as predictors of motivation outcomes in track and field: A self-determination theory perspective ´ from Bhatnagar and Karageorghis (2008) base their study on the same theory and explain the motives for sport participation. The authors claim that autonomous regulations are expected to correspond with positive outcomes and Reinecke and Meier also agree with this statement. When looking at mental health they claim that the self determination theory can be used as a explanation for mental health and user- centered communication approaches (p. 1203). This could be part of future researches, where the self determination theory is the main aspects. Questions like ´Is there a correlation between the need to satisfy the intrinsic motivation and media use?´ or ´To what extend does the need for autonomy and social connection influence media use?´ can be asked.
+
+Another theoretical aspect or more so, the basement of the main paper, is the division into technology-centered and user-centered approaches. This is one of the main argument of Reinecke and Meier when looking at CMC and MH. The correlation between CMC and MH is also the main topic in the paper of Meier, Domahidi and Günther (2020) where 20 years of study is retrieved through a systematic database search. Even though they didn’t divide into technology-centered and user-centered approaches, they did indeed mention different users in the context of how the access to information and communication technologies (ICTs) affects different marginalised approaches (p. 22). This is a expansion to the Reinecke and Meier study and could be an inspiration for further studies. Following questions could be asked: ‘How do the effects of CMC and social media on mental health vary across different populations (e.g. gender, age, cultural or social background), and how can future studies research these differences?’.
```

Comment:
Concerning the first question: Some studies have shown that experienced autonomy in a work space predominantly occur, if workers are motivated to work via digital media even outside of their working hours. This does not yet answer the question of cultural differences, it does however answer how varying attitudes toward digital media can affect mental health.